_posts/2023-05-09-ministry-of-economy-finance-uruguay-adopts-quarkus.adoc

```diff
@@ -287,28 +299,26 @@ This additional information helps our development team to debug errors reported
+It is possible to change the output format of the console log. This can be useful in environments where the output of the Quarkus application is captured by a service which processes and stores the log information for later analysis. In order to configure the JSON logging format, the `quarkus-logging-json` extension may be employed. See https://quarkus.io/guides/logging#json-logging.
 
-In our case, this allowed us to add some extra parameters to this JSON, that are easily picked up by fluentd and sent to Elasticsearch.
+In our case, this allowed us to add some extra parameters to the JSON log output, that are easily picked up by fluentd and sent to Elasticsearch.
 
 The `quarkus.log.console.json.exception-output-type=formatted` property is also used to send the stacktrace.
 
-WARNING: Docker has a limit size of 16K for logs (https://github.com/kubernetes/kubernetes/issues/52444[kubernetes/kubernetes#52444] and https://github.com/moby/moby/issues/34620[moby/moby#34620]). An error with a large stacktrace is going to generate a split log. The proposed fluentd community solution is using the fluent-plugin-concat. Keep in mind that there are edge cases with rotating file logs when it might fail. Another solution is sending the logs directly as mentioned in https://quarkus.io/guides/centralized-log-management.
+WARNING: Docker has a size limit of 16K for logs (https://github.com/kubernetes/kubernetes/issues/52444[kubernetes/kubernetes#52444] and https://github.com/moby/moby/issues/34620[moby/moby#34620]). An error with a large stacktrace is going to generate a split log. The proposed fluentd community solution is using the fluent-plugin-concat. Keep in mind that there are edge cases with rotating file logs when it might fail. Another solution is sending the logs directly as mentioned in https://quarkus.io/guides/centralized-log-management.
 
 
 === MDC (Mapped diagnostic context)
 
-The typical Java logging API lets you log the level, exception message and stacktrace. Quarkus uses JBoss logmanager under the hoods. Using `org.jboss.logmanager.MDC`, we have access to `MDC.put(String key, String value)` method that allows us to add key-value parameters.
+The typical Java logging API lets you log the level, exception message and stacktrace. Quarkus uses JBoss Log Manager under the hoods. Using `org.jboss.logmanager.MDC`, we have access to the `MDC.put(String key, String value)` method that allows us to add key-value parameters.
 In order to set this information, a `javax.ws.rs.container.ContainerRequestFilter` should be implemented.
 
-WARNING: If using RESTEasy Reactive, MDC context propagation works on Quarkus > 2.10.0
```

Comment:
Removed this as we are already at 2.16 and 3.0 is out so I don't think it's that useful to mention it.